src/module_2/module_2_groceries.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+import os
+import re
+import numpy as np
+from datetime import datetime
+
+
+DATA_PATH = "/home/unai/datasets/groceries/"
+TABLES = {"abandoned_carts": 0, "users": 1, "regulars": 2, "orders": 3, "inventory": 4}
+
+
+def main():
+    a = extract_data(DATA_PATH)
+    users_using_regulars(a[TABLES["regulars"]], a[TABLES["users"]])
```

Comment:
In general, these methods would better return values so we can foster them instead of just print results

Reply:
But in EDA is really useful return values?

Reply:
Answered in other comment, you may want in the future to use that information. Here you can just `_ = ` to discard the returned value and everything will work as it is but leaving the potential use of getting the return value for further analysis somewhere else